cacheops/query.py

```diff
@@ -440,9 +441,10 @@ def _post_save(self, sender, instance, **kwargs):
 
         # Invoke invalidations for both old and new versions of saved object
         old = _old_objs.__dict__.pop((sender, instance.pk), None)
+        using = kwargs['using']
         if old:
-            invalidate_obj(old)
-        invalidate_obj(instance)
+            invalidate_obj(old, using=using)
+        invalidate_obj(instance, using=using)
 
         # We run invalidations but skip caching if we are dirty
         if transaction_states[instance._state.db].is_dirty():
```

Comment:
What happens if `using != instance._state.db`?

Reply:
For post_save, instance._state.db is always equal to using.
https://github.com/django/django/blob/master/django/db/models/base.py#L754
(I work with django 1.9, it the same behavior).